TS03-Subset-Data-and-No-Data-Values.Rmd

```diff
@@ -165,7 +162,7 @@ qplot (datetime, prec,
 ## Missing Data - Dealing with data gaps
 
 ###Checking for No Data Values
-In the <a href="XXX" target="_blank"> Understand The Data</a>
+In the  [{{site.baseurl}} / LINKHERE](Lesson 01 - )
```

Comment:
@mguarinello @lwasser   I've noticed that we do switch back and forth between html and Rmd for how we are adding in links.  Does it matter which one we use?  Should we standardize?  I just don't want us editing different formats that work equally well.